.github/workflows/test.yml

```diff
@@ -0,0 +1,23 @@
+name: "Test"
+
+on:
+  - "push"
+
+jobs:
+  test:
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: "actions/checkout@v3"
+
+      - name: "Set up Python"
+        uses: "actions/setup-python@v4"
+        with:
+          python-version: "3.11"
+
+      - name: "Install dependencies"
+        run: |
+          pip install pandas pytest
```

Comment:
This is not the best practice, because it often means the dependency list is copied in multiple places. Better would be a dependencies file in the repo (better yet, a lockfile) that CI can use to install. But I wanted to be minimal :)